lib/safira_web/router.ex

```diff
@@ -25,10 +25,9 @@ defmodule SafiraWeb.Router do
       get "/user", AuthController, :user
 
       resources "/badges", BadgeController, only: [:index, :show]
-
       resources "/attendees", AttendeeController, except: [:create]
-
       resources "/referrals", ReferralController, only: [:show]
+      resources "/companies", CompanyController, only: [:index, :show]
```

Comment:
space missing :cry:

Reply:
Use bigger font

Reply:
At this point a newline between resources has been the default code style

Reply:
Always time for a change, in all seriousness since we have quite a handful of them I personaly think we should only separate the different ones

Reply:
ok sure, i think we need to review and define a code style, please just no meaningless parentheses, this is not lisp :cry: